### PR TITLE
feat: [SRM-15406]: Implementing the new UX for creating monitored services.

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Implementing the new UX for creating monitored services.

#### Screenshots
New:
New Note

Previous:
New Note

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.module.scss [hunk 1]
```diff
@@ -38,3 +38,10 @@
     color: var(--grey-400);
   }
 }
+
+.monitoredServiceContainer {
+  width: 874px !important;
+  border-radius: var(--spacing-small);
+  background: var(--primary-1) !important;
+  padding: 8px 16px !important;
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.module.scss:46]: use spacing CSS variables for spacing related properties

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.tsx [hunk 2]
```diff
@@ -99,112 +101,137 @@ export default function MonitoredServiceOverview(props: MonitoredServiceOverview
...
+              {getString('cv.monitoredServices.defineServiceEnvironment')}
+            </Text>
+            <Text lineClamp={1}>{getString('cv.monitoredServices.defineServiceEnvironmentDescription')}</Text>
+          </Layout.Vertical>
+          {isCDModule || !isSRMLicensePresentAndActive ? null : (
+            <FormInput.Select
+              name="type"
+              tooltipProps={{ dataTooltipId: 'monitoredServiceType' }}
+              items={MonitoredServiceTypeOptions}
+              label={'Service Type'}
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.tsx:123]: use label from YAML strings

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.module.scss (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.module.scss]: done.

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/MonitoredServiceOverview/MonitoredServiceOverview.tsx]: done.

**Review by deepesh-ui — DISMISSED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**